tracer/src/Datadog.Trace/Util/UriHelpers.cs

```diff
@@ -145,5 +145,33 @@ public static bool IsIdentifierSegment(string absolutePath, int startIndex, int
 
             return containsNumber;
         }
+
+        /// <summary>
+        /// Combines an absolute base <see cref="Uri"/> <paramref name="baseUri"/> with a path <paramref name="relativePath"/>.
+        /// If <paramref name="baseUri"/> includes a path component, this will be included in the final <see cref="Uri"/>.
+        /// The final <see cref="Uri"/> will always contain all path segments from both parameters.
+        /// </summary>
+        /// <example>The following calls all return the <see cref="Uri"/> <c>http://host/a/b/c</c>.
+        /// <code>Combine(new Uri("http://host/a/b"), "c");
+        /// Combine(new Uri("http://host/a/b"), "/c");
+        /// Combine(new Uri("http://host/a/b/"), "c");
+        /// Combine(new Uri("http://host/a/b/"), "/c");</code></example>
+        /// <param name="baseUri">The base <see cref="Uri"/>, which may or may not end with a <c>/</c>. </param>
+        /// <param name="relativePath">The relative path, which may or may not start with a <c>/</c>.</param>
+        /// <returns>The combined <see cref="Uri"/></returns>
+        public static Uri Combine(Uri baseUri, string relativePath)
```

Comment:
I thought I made a comment here about how a `relativePath` that starts with a path separator is expected to act as an absolute path instead. Where did the comment go? Did I forget to post it? 😕

Reply:
No, you posted it, and I did a big reply saying "I disagree" 😄 I'm guessing a force push has killed it, but that's a bit weird, I thought GitHub saved those 😕

Reply:
There it is, thanks! I missed your reply on that one. I need to get a handle on my github notifications 😞